src/libraries/System.Net.NetworkInformation/tests/FunctionalTests/NetworkInterfaceBasicTest.cs

```diff
@@ -53,6 +53,7 @@ public void BasicTest_AccessInstanceProperties_NoExceptions()
 
         [Fact]
         [PlatformSpecific(TestPlatforms.Linux)]  // Some APIs are not supported on Linux
+        [ActiveIssue(18090)]
```

Comment:
Nit: I believe we should be putting the full URL, @stephentoub changed all previous issues to do that as you can just CTRL+Click from the IDE.

Reply:
Should we then remove the int constructor from ActiveIssueAttribute?

Reply:
I think that's a good idea.  Are any other repos using it?

Reply:
I don't think so.

Reply:
I believe I was going to do that and completely forgot about it. Will put up a PR now.